src/rules/property-no-unknown/README.md

```diff
@@ -0,0 +1,119 @@
+# property-no-unknown
+
+Disallow unknown property.
+
+```css
+.foo { heigth: 100%; }
+/**    ↑
+ *  These properties */
+```
+
+This rule considers properties defined in the [CSS Specifications and browser specific properties](https://github.com/betit/known-css-properties#source) to be known.
+
```

Comment:
Can we get a paragraph below this that reads:

"This rule ignores variables (`$sass`, `@less`, `--custom-property`)."